windows/src/developer/TIKE/child/Keyman.Developer.UI.UfrmModelEditor.pas

```diff
@@ -130,13 +130,13 @@       TWordlists = class(TObjectList<TWordlist>)
 
     function GetProjectFile: TProjectFile; override;
 
+  public
```

Comment:
I didn't see this line on the original master PR

Reply:
Yeah, just fixes a compile hint (those functions should have been public because they are public in the parent class).